config/routes.rb

```diff
@@ -6,7 +6,9 @@
   get '/', to: 'welcome#index', as: 'landing'
   
   get '/register', to: 'users#new', as: 'register_user'
-  resources :users, only: [:show, :create] do
+  post '/register', to: 'users#create'
+
+  resources :users, only: [:show] do
     resources :discover, only: [:index]
   end
 end
```

Comment:
I wonder if we did something like this to try and make the routes a little more restful and not as much hand rolling? 
```ruby
root to: 'users#index'

get '/register', to: 'users#new', as: :register

resources :users, only: [:index, :new, :show, :create] do
   resources :discover, only: [:index]
end
```